app/src/main/res/values/strings.xml

```diff
@@ -440,6 +442,12 @@ Upload your first media by tapping on the add button.</string>
   <string name="ends_on">Ends on:</string>
   <string name="display_campaigns">Display campaigns</string>
   <string name="display_campaigns_explanation">See the ongoing campaigns</string>
+  <string name="in_app_camera_location_access_explanation">Allow the app to fetch location in case the camera does not record it. Some device cameras do not record location. In such cases, letting the app fetch and attach location to it makes your contribution more useful. You may change this any time from the Settings</string>
+  <string name="option_allow">Allow</string>
+  <string name="option_dismiss">Dismiss</string>
+  <string name="in_app_camera_needs_location">Please turn on location access from the Settings and try again. \n\nNote: The first upload may not have location due to limitations of device GPS.</string>
+  <string name="in_app_camera_location_permission_rationale">In-app camera needs location permission to attach it to your images in case location is not available in EXIF. Please allow the app to access your location and try again.\n\nNote: The first upload may not have location due to limitations of device GPS.</string>
```

Comment:
We specify "first upload" but I believe that isn't specificially true. The location won't be captured whenever GPS initialization fails. So, it might be worth tweaking the phrase to something like:

> Note: The upload may not have location if app is unable to retrieve location from device within a short interval.

Also, I believe it would be worth copying the note to `in_app_camera_needs_location` string too since the location request dialog won't be shown on devices that don't have permissions.

Reply:
> We specify "first upload" but I believe that isn't specificially true. 
Was it not supposed to be done to keep things simple for the users? I'm sorry, I think I misinterpreted something discussed over Zulip. Thank you for suggesting a better phrase, I'll change it :)